### PR TITLE
DAT-2507

### PR DESCRIPTION
**Pull request by RafLeszczynski**
Add sorting param to the get tile set collection request

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

**Comment by RafLeszczynski** [comment]
@rogatty thanks :-)

**Comment by nandy-andy** [comment]
Perfect! Tests :dancer: LGTM as well :+1:

File: api/v1/tile_set.config.js [hunk 1]
```diff
@@ -62,6 +62,16 @@ module.exports = {
...
+		desc: {
+			column: 'created_on',
+			direction: 'desc'
+		},
+		asc: {
+			column: 'created_on',
+			direction: 'asc'
+		}
+	},
 	searchLimit: 50,
```
Comment (rogatty) [inline api/v1/tile_set.config.js]: Is this being used anywhere?

Comment (RafLeszczynski) [inline api/v1/tile_set.config.js]: yes, it's used as max `LIMIT` for `SELECT` query with search term. If a limit sent as request param is greater than `searchLimit ` the `searchLimit ` will be used

**Pull request merged**